src/App.tsx

```diff
@@ -47,6 +63,12 @@ const App = (): JSX.Element => {
             </Route>
             <Route path="*" element={<NoMatch />} />
           </Routes>
+          <ReactQueryDevtools initialIsOpen />
```

Comment:
```suggestion
          <ReactQueryDevtools initialIsOpen position="top-left" />
```

The default position of "bottom-left" makes zoom controls difficult to access, at least on my screen size. With our current UI, any other position would be less intrusive. I've gone with "top-left" to future-proof against the potential removal of the fixed action panel.

 We may want to do this for the production component as well.

Reply:
OK.